keyboards/moonlander/matrix.c

```diff
@@ -216,9 +220,12 @@ void matrix_power_up(void) {
         ML_LED_1(false);
         ML_LED_2(false);
         ML_LED_3(false);
+
+        #ifndef HALFMOON
```

Comment:
This shouldn't be needed, IMO. It's setting part of a variable that just doesn't get used 
```suggestion
```